src/main/java/org/folio/circulation/resources/RequestFromRepresentationService.java

```diff
@@ -535,8 +539,8 @@ private Result<Request> validateAbsenceOfItemLinkInTlr(Request request) {
       Map<String, String> errorParameters = new HashMap<>();
       errorParameters.put("itemId", itemId);
       errorParameters.put("holdingsRecordId", holdingsRecordId);
-      return failedValidation("Attempt to create TLR request linked to an item",
-        errorParameters);
+      return failedValidation("Cannot create a title level page request " +
+          "for this instance ID with no item ID", errorParameters, INSTANCE_HAS_NO_ITEM_ID);
```

Comment:
Error code looks a bit misleading. The problem here is that it is impossible to create a page TLR without item ID. Can we think of a better error code to reflect that?